### ISSUE TITLE
Emptying trash with multiple folders and files fails

### ISSUE THREAD

**Issue by ghost** [cryptpad/cryptpad#1344]
### Description

Emptying the trash with the following content:

One folder, "test1" containing 652 files and another folder, "test2", which contain 199 files and another folder "test3", containing 17 files. Each is populated by empty files, totalling 862 items.

```
test1
├── 652_files
└── test2
    ├── 199_files
    └── test3
        └── 17_files
```

### Steps to reproduce

1. Create 3 test folders on your computer
2. Fill them with empty files
3. Upload them on CryptPad
4. Trash "test1" folder
5. Try to empty the trash, either Destroying or Removing the content
6. Wait one minute
7. Watch it fails
8. Get disconnected from your drive until you reload the page

See the following screen recording, error pops up at 00:56:

https://github.com/cryptpad/cryptpad/assets/41020854/a8c11afe-7408-40a0-8c2a-7fd1d36ce122

### Relevant logs

```shell
Object { type: "DISCONNECTED", message: '[1028,"MSG","b9d464e048098139fc6f6d09885e84c8","NPJHpdK7WeTRHX6y/6pYUP2AQdITMIawvSr+HeAJJUofstp5588h4810Mv8XGuQElZSV49yDm2YjwJb8AhWBBzNhdW1rcXV0ckM5dEg2UThjaVNrZGJ5clF1U3A5UzR0fHVBeXdwaC9aSnl2NGxMc1BkMzhtOUtmMFdERmh4dFovU0xaMUVSYktPamZIN0dJbmUraW1WNS9naVVIL3d6TnhoUDVyNWRmbUoxbG56eEdXSDdVL3h5VjR6dGRURlZKeitIbDYrVVRmNGZ3YXZlME5oTXRnbTBJVUVLcE5kL0lkNmVVc0F6T0RLcDE3SUVyWUlpcGhkb0ZKWko1dGJEdDk5cVI5akZHQlZ1c1lpL2ZRTXhUVE5CenpmbVNaV2xxL1NJNXBURy81VWlBTlRTOVVjUGRyMm8vQzRSa3NMaWhsZlE9PQ=="]' }
​
message: '[1028,"MSG","b9d464e048098139fc6f6d09885e84c8","NPJHpdK7WeTRHX6y/6pYUP2AQdITMIawvSr+HeAJJUofstp5588h4810Mv8XGuQElZSV49yDm2YjwJb8AhWBBzNhdW1rcXV0ckM5dEg2UThjaVNrZGJ5clF1U3A5UzR0fHVBeXdwaC9aSnl2NGxMc1BkMzhtOUtmMFdERmh4dFovU0xaMUVSYktPamZIN0dJbmUraW1WNS9naVVIL3d6TnhoUDVyNWRmbUoxbG56eEdXSDdVL3h5VjR6dGRURlZKeitIbDYrVVRmNGZ3YXZlME5oTXRnbTBJVUVLcE5kL0lkNmVVc0F6T0RLcDE3SUVyWUlpcGhkb0ZKWko1dGJEdDk5cVI5akZHQlZ1c1lpL2ZRTXhUVE5CenpmbVNaV2xxL1NJNXBURy81VWlBTlRTOVVjUGRyMm8vQzRSa3NMaWhsZlE9PQ=="]'
​
type: "DISCONNECTED"
​
<prototype>: Object { … }
chainpad-netflux.js:515:37
    send https://cryptpad.fr/components/chainpad-netflux/chainpad-netflux.js?ver=5.5.0-g:515

Object { type: "DISCONNECTED", message: '[1029,"MSG","b9d464e048098139fc6f6d09885e84c8","vaC0Jp3C6bl0AAduVn2frI7NnUVZ/jEuow5n/Th3tpV5uh6S3Aa64kwIoq5GScWlKuXSirk+kHP+uRUKhxtLD09KS2VSYWpBUm95dzUzdmY4Q1lnbXRMS0MwcHVIanhzfHdKbThTZnJvcXU3Tm02TkRVT3NEcWFvYU5FRlZyMVBMMEFQb1hyZytidktyWVpiV1hNWGhrOGJhT2lnblJQS09HNkZTZmdxajYzSHdqVTFpbm1sT1VVVHgwTnZiWmUvdlZkREVjL0tLU1o0ejV3Zk9aODBOWEQxUGx4bGd4T1FQU01DOVNKUTJyZ0xya2JTek5HQ3o0UnExZzZ4Ty9CVTZKZlVLUWRNT2FZLzJWNzBUVExwNkhCY1o4UzE4ZWJzdkY4OUYxVEFiNTBoSUZkNjJ2V2JzYzhZS0x4b2dqOEttYXc9PQ=="]' }
​
message: '[1029,"MSG","b9d464e048098139fc6f6d09885e84c8","vaC0Jp3C6bl0AAduVn2frI7NnUVZ/jEuow5n/Th3tpV5uh6S3Aa64kwIoq5GScWlKuXSirk+kHP+uRUKhxtLD09KS2VSYWpBUm95dzUzdmY4Q1lnbXRMS0MwcHVIanhzfHdKbThTZnJvcXU3Tm02TkRVT3NEcWFvYU5FRlZyMVBMMEFQb1hyZytidktyWVpiV1hNWGhrOGJhT2lnblJQS09HNkZTZmdxajYzSHdqVTFpbm1sT1VVVHgwTnZiWmUvdlZkREVjL0tLU1o0ejV3Zk9aODBOWEQxUGx4bGd4T1FQU01DOVNKUTJyZ0xya2JTek5HQ3o0UnExZzZ4Ty9CVTZKZlVLUWRNT2FZLzJWNzBUVExwNkhCY1o4UzE4ZWJzdkY4OUYxVEFiNTBoSUZkNjJ2V2JzYzhZS0x4b2dqOEttYXc9PQ=="]'
​
type: "DISCONNECTED"
​
<prototype>: Object { … }
chainpad-netflux.js:515:37
    send https://cryptpad.fr/components/chainpad-netflux/chainpad-netflux.js?ver=5.5.0-g:515
```

### Expected behavior

The trash bin should be able to be emptied, even if it contains hundreds of items.

### Workaround tried

Deleting the last folder, "test3" first, which contains only 17 files works by using the "Remove" button when right-click on it.

<img width="513" alt="Screenshot 2023-12-14 at 18 52 00" src="https://github.com/cryptpad/cryptpad/assets/41020854/1e97e03b-8c5f-40d0-801b-75444f4ef241">

However, the page don't update itself. You have to reload it to see the folder disappeared. Web browser console logs gives us:

```
Array [ "remove", "editable" ]
​
0: "remove"
​
1: "editable"
​
length: 2
​
<prototype>: Array []
drive-ui.js:4730:21
```

Deleting the second folder, "test2" first, which contains 199 files works by using the "Remove" button when right-click on it. However, the page don't update itself. You have to reload it to see the folder disappeared. Web browser console logs gives us the same message as before.

However, trying to delete the "test1" folder, containing 652 files fails.

<img width="758" alt="Screenshot 2023-12-14 at 18 52 51" src="https://github.com/cryptpad/cryptpad/assets/41020854/e41e9e11-23aa-47c1-819a-404f10f6db21">

Giving the following error message:

```
{
  "type": "EJOINED",
  "message": "a975e12bfdb64f24725fae5e1c2f581b"
}
```

To summarize, it's not completely working. Also, it's not satisfying that we aren't able to Destroy the files and can only Remove them with the right click contextual menu within the trash bin.

### What opreating system are you using?

Linux/BSD/UNIX, Microsoft Windows, Apple macOS

### What browsers are you seeing the problem on?

Mozilla Firefox, Google Chrome

### Extensions

Not relevant

### Version

5.5.0
